Civi/Financeextras/Hook/BuildForm/RefundCreditNotePaymentInformation.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Civi\Financeextras\Hook\BuildForm;
+
+class RefundCreditNotePaymentInformation {
+
+  /**
+   * @param \CRM_Financial_Form_Payment $form
+   */
+  public function __construct(private \CRM_Financial_Form_Payment $form) {
+  }
+
+  public function handle() {
+    if ($this->form->elementExists('check_number')) {
+      $element = $this->form->getElement('check_number');
+      $element->setLabel('Cheque Number');
```

Comment:
@shahrukh-compuco, we only want to use `Cheque` based on the current language.

can we use ts() function.